.gitignore

```diff
@@ -1,30 +1,48 @@
 # See https://help.github.com/articles/ignoring-files for more about ignoring files.
 #
-# If you find yourself ignoring temporary files generated by your text editor
-# or operating system, you probably want to add a global ignore instead:
-#   git config --global core.excludesfile '~/.gitignore_global'
+# Temporary files generated by your text editor or operating system
```

Comment:
This is me getting our .gitignore in line with GBL's and putting all our customizations after there's. This should hopefully help keep it in sync with any of their forthcoming changes.